openapi/components/schemas/Tracking/ApiTracking.yaml

```
@@ -89,6 +89,19 @@ properties:
         description: Specifies if the API request is completed by a Rebilly Support Team member.
         type: boolean
         x-sortable: true
+  organization:
+    type: object
+    description: Organization of the user who made the request.
```

user may have many organizations so the description is not quite correct

```suggestion
    description: Organization in scope of which the request was made.
```

The suggestion is also difficult to read, but demonstrates the idea

---

How about `Organization from which the request is made.`